src/EtherFiRestaker.sol

```diff
@@ -39,8 +40,8 @@ contract EtherFiRestaker is Initializable, UUPSUpgradeable, OwnableUpgradeable,
     mapping(address => TokenInfo) public tokenInfos;
     
     EnumerableSet.Bytes32Set private withdrawalRootsSet;
-    mapping(bytes32 => IDelegationManager.Withdrawal) public withdrawalRootToWithdrawal;
 
+    IRewardsCoordinator public immutable rewardsCoordinator;
```

Comment:
I believe removing the mapping and then declaring this new variable as immutable causes a storage issue since the immutable variable won't be in storage. There will be a conflict for the next storage variable we add in a future upgrade

Reply:
makes sense. we can keep the mapping variable putting `DEPRECATED_` prefix

Reply:
fixed